src/components/dashboard/FaceVerification/hooks/useFaceTecSDK.js

```diff
@@ -22,17 +22,25 @@ const log = logger.child({ from: 'useFaceTecSDK' })
  *
  * @return {void}
  */
-export default ({ onInitialized = noop, onError = noop }) => {
+export default eventHandlers => {
+  // parse options
+  const { onInitialized = noop, onError = noop } = eventHandlers || {}
+
+  // state vars
+  const [initialized, setInitialized] = useState(false)
+  const [lastError, setLastError] = useState(null)
```

Comment:
as we provining also an prop-style interface in addition to callback-style
let's also will return an error from the hook